keylime/api_version.py

```diff
@@ -34,7 +37,12 @@ def all_versions():
 
 def is_supported_version(v):
     v_obj = version.parse(str(v))
-    return v_obj.base_version in VERSIONS
```

Comment:
I would still prefer we explicitly check if it's in the `VERSIONS` list. If not, we could fall back to a check that it's only a little bit more than the `CURRENT_VERSION`, maybe just 1 or 2 minor versions different. And if that is the case, we issue a warning saying the agent's version is from the future and we can't guarantee compatibility, or something like that.

Reply:
Or maybe we should leave `is_supported_version()` alone and create a new method named something like `is_slightly_future_version()` or something similar. Than we can be very clear that a slight future version is not supported, but we'll allow the request to go through anyway.